packages/flutter_tools/lib/src/web/compile.dart

```diff
@@ -51,10 +53,12 @@ Future<void> buildWeb(FlutterProject flutterProject, String target, BuildInfo bu
     result = false;
   } finally {
     status.stop();
+    sw.stop();
   }
   if (result == false) {
     throwToolExit('Failed to compile $target for the Web.');
   }
+  flutterUsage.sendTiming('build', 'dart2js', Duration(milliseconds: sw.elapsedMilliseconds));
```

Comment:
for dart2js you want to ensure that buildInfo.isRelease is true, otherwise this is a ddc build at the time will vary based on what is in the cache.

Reply:
Done.